vector/src/main/java/im/vector/app/features/settings/VectorPreferences.kt

```diff
@@ -415,7 +415,7 @@ class VectorPreferences @Inject constructor(private val context: Context) {
      * @return true true if the avatar and display name events should be shown in the messages list.
      */
     fun showAvatarDisplayNameChangeMessages(): Boolean {
-        return defaultPrefs.getBoolean(SETTINGS_SHOW_AVATAR_DISPLAY_NAME_CHANGES_MESSAGES_KEY, true)
+        return defaultPrefs.getBoolean(SETTINGS_SHOW_AVATAR_DISPLAY_NAME_CHANGES_MESSAGES_KEY, false)
```

Comment:
I unsintalled the application and install a new one by using your branch, for unknown reasons these 2 preferences were enabled after a new connexion. This looks like the default values is overridden somewhere else
Can you please test on your side?

Reply:
I think I found where (see https://github.com/tchapgouv/tchap-android-v2/pull/277/files#diff-f71c85b34c236d34da29a61fa263f190f0e5b7664a531e84741cbc90c003d234L109 and https://github.com/tchapgouv/tchap-android-v2/pull/277/files#diff-f71c85b34c236d34da29a61fa263f190f0e5b7664a531e84741cbc90c003d234L115)